.envrc

```diff
@@ -0,0 +1 @@
+dotenv
```

Comment:
Can this file just be removed?

Reply:
Okay, I didn't understand this was a function/macro/command supported by direnv.  Based on the description of the pull request I thought this file was going away, and this was your attempt at clearing it out to be removed in the future.  Maybe add a comment about what the purpose of the `dotenv` is in this file?